test/test_kernel.cpp

```diff
@@ -33,15 +33,13 @@ TEST_CASE(
   args.emplace_back(KernelArg(&datasize));
 
   Headers headers;
```

Comment:
I guess I could have just removed the Headers headers as well, but seems fine to me like this.